src/components/Calculator.js

```diff
@@ -1,42 +1,63 @@
 import React from 'react';
 import Button from './Button';
```

Comment:
- [ ] Here, inside Calculator.js is where you could import the calculte.js file, please do this. 🐶

Reply:
Sorry, but why would I have to import the calculate.js file there if I don't use it in Calculator.js? I use the calculate function in Button.js